spec/unit/views/components/index_table_for_spec.rb

```diff
@@ -42,5 +42,65 @@
         end
       end
     end
+
+    context 'when creating an index column' do
+      let(:base_collection) do
+        posts = [
+          Post.new(title: 'First Post', starred: true),
+          Post.new(title: 'Second Post', starred: true),
+          Post.new(title: 'Third Post', starred: true),
+          Post.new(title: 'Fourth Post', starred: true)
+        ]
+        Kaminari.paginate_array(posts, limit: 2)
+      end
+      let(:collection) do
+        base_collection.page(1)
+      end
+
+      let(:table) do
+        render_arbre_component assigns, helpers do
+          insert_tag(ActiveAdmin::Views::IndexAsTable::IndexTableFor, collection, {sortable: true}) do
+            index_column
+          end
+        end
+      end
+
+      context 'creates a table header based on the index column' do
+        let(:header) do
+          table.find_by_tag('th').first
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.